src/Components/test/testassets/ComponentsApp.Server/Startup.cs

```diff
@@ -29,10 +29,13 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             }
 
             app.UseStaticFiles();
-            app.UseRouting(builder =>
+
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints =>
```

Comment:
I'm sure this will be documented, but TBH it's not clear to me from looking at the code how `UseRouting` and `UseEndpoints` do different jobs. I can't guess why both are needed or what kinds of things go into each.

Not saying this is a problem, just that we'll need to have good clear explanations.

Reply:
yes, already writing the blog post.